modules/api/src/main/java/net/countercraft/movecraft/util/Tags.java

```diff
@@ -71,4 +71,22 @@ public static NamespacedKey keyFromString(@NotNull String string){
             return null;
         }
     }
+
+    /**
+     * Gets a set of materials from the specific string
+     * This is intended to be used to parse material names or tags from a config file
```

Comment:
This should describe what the method does, such as saying how it first checks for a tag, and than falls back to a material.